src/SFA.DAS.ApplyService.InternalApi/Controllers/ExperienceAndAccreditationController.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.ApplyService.Application.Apply.Roatp;
+using SFA.DAS.ApplyService.InternalApi.Infrastructure;
+
+namespace SFA.DAS.ApplyService.InternalApi.Controllers
+{
+    [Authorize]
+    public class ExperienceAndAccreditationController : Controller
+    {
+        private readonly IInternalQnaApiClient _qnaApiClient;
+
+        /// <summary>
+        /// Returns trading name for an application
+        /// </summary>
+        /// <param name="qnaApiClient"></param>
+        public ExperienceAndAccreditationController(IInternalQnaApiClient qnaApiClient)
+        {
+            _qnaApiClient = qnaApiClient;
+        }
+
+        [HttpGet("/Gateway/{applicationId}/OfficeForStudent")]
+        public async Task<string> GetOfficeForStudents(Guid applicationId)
+        {
+            var websiteNamePage = await _qnaApiClient.GetPageBySectionNo(applicationId, 
+                RoatpWorkflowSequenceIds.YourOrganisation, 
+                RoatpWorkflowSectionIds.YourOrganisation.ExperienceAndAccreditations, 
+                RoatpWorkflowPageIds.ExperienceAndAccreditations.OfficeForStudents);
+
+            return websiteNamePage?.PageOfAnswers?.SelectMany(a => a.Answers)?
```

Comment:
Wondering if we could make this a function in a base class to avoid repeated code across multiple controllers?

Reply:
possibly.  I like the idea of a base class when you know it's the same sequence and section, and only the page changes.  Let's leave it as is, but establish a pattern whenever you see more than 2? That sound good?

Reply:
is it a good idea to return a string from a REST API? Normally it would be JSON (hence like the Answer object being used in QnA). Also it limits you somewhat if you need to expand the returned value into a object

Reply:
We return stuff like that when it's not a primitive.  It's just these ones are just a string....

We'll have other logical entities if needed.  I think Dave's done stuff like that in Criminal Compliance